config/sync/core.extension.yml

```diff
@@ -37,7 +37,6 @@ module:
   node: 0
   options: 0
   origins_common: 0
-  origins_layouts: 0
```

Comment:
Why was this uninstalled?

Reply:
I was trying to disable layout builder to see if it was conflicting with the display suite module. Forgot to turn it back on. Enabled again